src/main/resources/templates/home.html

```diff
@@ -1,28 +1,77 @@
 <!DOCTYPE html>
-<html lang="en">
+<html xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
-    <title>Home</title>
-    <script src="https://unpkg.com/htmx.org@2.0.3"
-            integrity="sha384-0895/pl2MU10Hqc6jd4RvrthNlDiE9U1tWmX7WRESftEDRosgxNsQG/Ze9YMRzHq"
-            crossorigin="anonymous"></script>
+    <title th:text="${@environment.getProperty('spring.application.name')}">Nome del Progetto</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap" rel="stylesheet">
 </head>
 <body>
-<h1>Enter a word</h1>
-<form>
-    <label for="word1">Word</label>
-    <input type="text" id="word1" name="word1">
-    <button type="submit">Submit where? Dont know...</button>
-    <button type="reset">Reset</button>
-</form>
-
-<span
-        hx-get="/currentEmoji"
-        hx-trigger="every 4s"
-        hx-target="this"
-        hx-swap="innerHTML">
-  /ᐠ - ˕ -マ Ⳋ
-</span>
 
+<!-- Header (contiene un pulsante con un menù a tendina, il logo del progetto e il nome) -->
+<header class="header">
+    <button class="menu-button" onclick="toggleMenu()">
+        &#9776; <!-- Icona del menu -->
+    </button>
+    <div class="logo-container">
+        <img src="logo.png" alt="Logo" class="logo">
+        <h1 th:text="${@environment.getProperty('spring.application.name')}"></h1>
+    </div>
+</header>
+
+<!-- Menu a tendina (da modificare con i veri nomi delle pagine)-->
+<nav id="dropdown-menu" class="dropdown-menu">
+    <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Assets</a></li>
+        <li><a href="#">Simulation</a></li>
+        <li><a href="#">History</a></li>
+    </ul>
+</nav>
+
+<!-- Sezione di Benvenuto -->
+<div class="content">
+    <h2 class="animate-slide-in-left">Benvenuti nel nostro portale di simulazione avanzata!</h2>
+    <p class="animate-slide-in-left">Il nostro software offre una piattaforma innovativa e intuitiva per simulare e monitorare il funzionamento di vari asset, connettendo dispositivi produttori e consumatori di energia in un ambiente virtuale dettagliato e configurabile. Sviluppato con tecnologia open source e senza costi aggiuntivi, il nostro prodotto consente agli utenti di sperimentare e ottimizzare sistemi complessi come Smart Grid, automazione industriale, e dispositivi per Smart Home.</p>
```

Comment:
Pulire un po le righe in modo che il testo non vada sia lungo 4 schermi. Cerca di scrivere codice che si vede in un unica schermata. In questo caso puoi avere un tag che si apre a riga 10 e si chiude a riga 15 e metti il testo in righe in mezzo. 

Fai questa modificha in altre parti di questa pagina.